### ISSUE TITLE
Mutation range inconsistent with mutation-testing-schema?

### ISSUE THREAD

**Issue by Aimless321** [stryker-mutator/editor-plugins#11]
I have been locally running MSP using stryker-js (https://github.com/stryker-mutator/stryker-js/pull/5086).
While testing I noticed that it's currently not working if i pass the a result from `discover` into `mutationTest`.

For example turning this mutant:
```json
{
  "files": {
    "src/utils/file-utils.ts": {
      "language": "typescript",
      "source": "some code",
      "mutants": [
        {
          "id": "2",
          "location": {
            "start": {
              "line": 8,
              "column": 32
            },
            "end": {
              "line": 8,
              "column": 64
            }
          },
          "mutatorName": "ObjectLiteral",
          "replacement": "{}",
        }
      ]
    }
  }
}
```

Into `src/utils/file-utils.ts:8:32-8:64` and passing it to `mutationTest` doesn't test any mutants.

Could this be caused by the [mutation-testing-schema location](https://github.com/stryker-mutator/mutation-testing-elements/blob/master/packages/report-schema/src/mutation-testing-report-schema.json#L324C1-L327) to have an exclusive end, and the MSP an inclusive?

**Comment by jaspervdveen** [comment]
Good catch! I believe this originated from the StrykerJS mutate config option, where it is inclusive. https://stryker-mutator.io/docs/stryker-js/configuration/#mutate-string
It think we need to change MSP to align the location with the MTE location.
